Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -138,10 +150,14 @@ export default class CovidTreatmentAssessmentComponent extends Vue {
     }
 
     private get patientFullName(): string {
+        this.covidTreatmentAssessmentRequest.firstName = this.patient.firstname;
+        this.covidTreatmentAssessmentRequest.lastName = this.patient.lastname;
```

Comment:
We shouldn't have side effects in getters; these values can be set right before we submit the request.

Reply:
@MikeLyttle but there is no issue also if we create a separate function?

Reply:
Instead of merging with `setAddressRequest()`?  Yeah, they could be populated in another function so long as it was called right before we submit the request.